### DISCUSSION TITLE
QFieldCloud : project status is busy for more than 2 hours

### DISCUSSION THREAD

**Discussion by jpdupuy** [opengisch/QField#2602] [Q&A]
Hello,

Although I have pushed my project 2 hours ago, the project status is still busy (red icon alongside project name).
The project is made of 385 files (1 qgs + 384 very small SVG files around 1K to 8K each)

QFieldSync also show a red icon for this specific project.
I have published the project with less SVG files, and the project status became green after 15 minutes.

Could you please check why it is taking so long ? or if definitly stuck ?

**Comment by suricactus** [comment]
As already discussed in https://github.com/opengisch/qfieldcloud/issues/299, the fix will be deployed by tomorrow. Thanks for letting us know the issue.

**Reply by jpdupuy** [reply]
Looks fine now. Thanks for the quick fix
